extension/source/QuillPage/components/WorkflowNumbers.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import Range from '../../helpers/Range';
+
+const WorkflowNumbers: React.FunctionComponent<{
+  current: number;
+  max: number;
+}> = ({ current, max }) => (
+  <div className="workflow-numbers quick-row">
+    {Range(max).map((i) => (
+      <div className="quick-row" key={i}>
+        <div
+          className={i + 1 <= current ? 'number highlight' : 'number'}
+          onClick={() => onSelect(i + 1)}
```

Comment:
```suggestion
          onClick={() => onSelect(i)}
```
onSelect already adds 1 to the page number
`?p=${pageIndex + 1}`

Reply:
Done